buildenv/jenkins/common/build.groovy

```diff
@@ -326,8 +326,63 @@ def archive_sdk() {
     }
 }
 
+/*
+* When crash occurs in java, there will be an output in the log IEATDUMP success for DSN='JENKINS.JVM.JENKIN*.*.*.X&DS'
+* The X&DS is a macro that is passed to the IEATDUMP service on z/OS
```

Comment:
The following lines are sentences, but need to have a period at the end of each one.